### DISCUSSION TITLE
before i was using yolov5, but now i need to use yolov8

### DISCUSSION THREAD

**Discussion by hitrate77** [ultralytics/ultralytics#8169] [General]
i don't have any idea, how to put this in yolov8

i have tried once with yolov8, but it didn't work 

this one is my yolov5 imp. code, 

model = torch.hub.load(r'C:\Users\911856\Desktop\GOOD\yolov5' , 'custom', path= r'C:\Users\911856\Desktop\GOOD\dudal.engine',source='local',force_reload=True).cuda()

what can i do to make it work for yolov8 

i know its stupid to ask..

**Comment by pderrenger** [comment]
Hey there! Not a stupid question at all, we're here to help! 😊 For YOLOv8, you can load your custom model like this:

```python
from ultralytics import YOLO

# Load your custom model
model = YOLO('path/to/your/custom_model.pt').cuda()
```

Make sure to replace `'path/to/your/custom_model.pt'` with the actual path to your YOLOv8 model file. If you encounter any issues, feel free to reach out again! 👍
